bin/protonPack

```diff
@@ -14,13 +14,21 @@ const { getPort, getPublicPath, findPort } = require('../webpack/helpers/source'
 
 const is = (command) => argv._.includes(command);
 
-const getAppMode = () => {
-    // Many flags appMode creates an array, take the last one.
-    if (Array.isArray(argv.appMode)) {
-        const { length, [length - 1]: last, first } = argv.appMode;
-        return last || first || 'bundle';
+const getSingleArgument = (argument) => {
+    // Many flags creates an array, take the last one.
+    if (Array.isArray(argument)) {
+        const { length, [length - 1]: last, first } = argument;
+        return last || first;
     }
-    return argv.appMode || 'bundle';
+    return argument;
+};
+
+const getFeatureFlags = () => {
+    return getSingleArgument(argv.featureFlags) || '';
```

Comment:
`featureFlags` is always a string ? doing `--featureFlags=monique,jeanne` won't create an array `['monique', 'jeanne'']` ?

Reply:
yeah it's just to be safe with the issue we had with the `appMode` you remember? if it is accidentally passed `--featureFlags=monique,jeanne --api=blue --featureFlags=blabla` then it would become an array